src/AggregateSpecsPlugin.ts

```diff
@@ -14,10 +14,48 @@ const DOUBLE_PRECISION_OID = "701";
 const INTERVAL_OID = "1186";
 const MONEY_OID = "790";
 
+const NUMERIC_CATEGORY = "N";
+const DATE_TIME_CATEGORY = "D";
+const NETWORK_ADDRESS_CATEGORY = "I";
+const STRING_CATEGORY = "S";
+
+const ENUM_TYPE = "p";
+
+const SUM_DEFAULTS = Object.freeze({
+  [MONEY_OID]: "0::money",
+  [INTERVAL_OID]: "INTERVAL '0 DAY'",
+});
+
+function isNumberLike(pgType: PgType): boolean {
+  return pgType.category === NUMERIC_CATEGORY;
```

Comment:
Using categories to filter can cause unintended consequences; for example `regproc`, `regoper`, etc etc are all numeric category; but it doesn't make sense to run aggregates such as average, sum, stddev on them. Please go back to explicit filtering - we can explicitly add new types to the filters in future without casting the net so wide as to catch nonsense things.